### PR TITLE
Issue #112 

### PR DESCRIPTION
**Pull request by fbbunny**
pictures moved from About to Members page

### CODE REVIEW

**Comment by goranche** [comment]
this is related to #112

**Comment by anuschka** [comment]
It works great :+1: and there is no overlap with the footer area. Now we just need to figure out how to get the pictures in and out of the table "userprofiles" :smile:

**Comment by anuschka** [comment]
@fbbunny: Your instructions on FCB to get the code from somebody else github branch and try it locally also work awesome :+1:

File: static/css/style.css [hunk 1]
```diff
@@ -106,6 +106,11 @@ code {
 	position: relative;
 }
 
+h1 {
+	text-align: center;
+	position: relative;
+}
+
 /* about page*/
 img{
```
Comment (ercchy) [inline static/css/style.css]: Every img on the site is going to have 100 width and height.

**Comment by ialja** [comment]
@fbbunny Now you can figure out how to display Members from the database, using models and all that fancy stuff. See: http://tutorial.djangogirls.org/en/dynamic_data_in_templates/README.html

**Comment by fbbunny** [comment]
@ialja my time, heart and soul will be dedicated to this this afternooon and evening ;)

**Comment by ialja** [comment]
@fbbunny Oh, how I love hearing these words! :grin: :heart: You go, Cat! :cat:

**Comment by goranche** [comment]
> my time, heart and soul will be dedicated to this this afternooon and evening

you deserve a :rose: and a :chocolate_bar:

**Comment by PettraV** [comment]
@fbbunny the description of your dedication is mind blowing, love ti! and you did a great job with members :+1:

**Pull request merged**